xarray/coding/variables.py

```diff
@@ -662,8 +662,8 @@ def encode(self, variable: Variable, name: T_Name = None) -> Variable:
                             SerializationWarning,
                             stacklevel=10,
                         )
-                    data = np.around(data)
-                data = data.astype(dtype=dtype)
+                    data = duck_array_ops.around(data)
```

Comment:
```suggestion
                    data = duck_array_ops.round(data)
```
Should try to phase out `around` while at it.